spring-session/src/test/java/org/springframework/session/jdbc/config/annotation/web/http/JdbcHttpSessionConfigurationTests.java

```diff
@@ -181,6 +207,17 @@ public PlatformTransactionManager transactionManager() {
 	}
 
 	@Configuration
+	static class CustomTableNameSetConfiguration extends JdbcHttpSessionConfiguration {
+
+		@Autowired
+		public void customize() {
```

Comment:
I guess it's taken from what I've done in Spring Boot. I agree an `@Autowired` method with no parameter is not really a good idea.